FileStorage/Controllers/FileController.cs

```diff
@@ -163,5 +164,53 @@ public HttpResponseMessage CreateFile(string filepath)
                 await content.CopyToAsync(fs);
             }
         }
+
+        [HttpGet]
+        [Route("api/File/{*filepath}")]
+        public HttpResponseMessage GetFile(string filepath)
+        {
+            HttpResponseMessage result = new HttpResponseMessage();
+            result.StatusCode = System.Net.HttpStatusCode.InternalServerError;
+
+            long length = 0;
+            try
+            {
+                if (new DirectoryInfo("files").Exists == false)
+                {
+                    return result;
+                }
+
+                string folderPath = Path.GetDirectoryName(filepath);
+
+                if (Directory.Exists(Path.Combine("files", folderPath)) == false)
+                {
+                    result.StatusCode = System.Net.HttpStatusCode.NotFound;
+                }
+                else
+                {
+                    if (File.Exists(Path.Combine("files", filepath)) == false)
+                    {
+                        result.StatusCode = System.Net.HttpStatusCode.NotFound;
+                    }
+                    else
+                    {
+                        var stream = new FileStream(Path.Combine("files", filepath), FileMode.Open,
+                                FileAccess.Read, FileShare.None);
+                        result.Content = new StreamContent(stream);
+
+                        result.Headers.Add("Server", System.Environment.MachineName);
```

Comment:
We should add more headers here: https://msdn.microsoft.com/en-us/library/azure/dn194274.aspx => Response Headers and also I would like us to support ranges